File: BroadcastForClub/PostResult/DAL/DatabaseConnection.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Data;
using System.Data.SqlClient;
using System.IO;

namespace PostResult.DAL
{
    class DatabaseConnection
    {
        #region Variables
        public SqlConnection sqlConn;
        public SqlCommand sqlComm;
        string servername = "";
        string databasename = "";
        string username = "";
        string password = "";
        #endregion

        #region Private Methods
        private void ReadConnecntionStringFile()
        {
            try
            {
                string sysDir = "";
                string connectionString = "";
                sysDir = AppDomain.CurrentDomain.BaseDirectory;
                connectionString = sysDir + "\\TextFile\\ConnectionString.inf";

                if (File.Exists(connectionString))
                {
                    TextReader tr = new StreamReader(connectionString);
                    using (tr)
                    {
                        servername = tr.ReadLine();
                        databasename = tr.ReadLine();
                        username = tr.ReadLine();
                        password = tr.ReadLine();
                    }
                }
            }
            catch (Exception ex)
            {
                throw ex;
            }
        }
        #endregion

        #region Public Methods
        public void DatabaseConn(string procName)
        {
            try
            {
                sqlConn = new SqlConnection();
                sqlComm = new SqlCommand();
                this.ReadConnecntionStringFile();
                sqlConn.ConnectionString = "Address=" + servername + ";database=" + databasename + ";user id=" + username + ";pwd=" + password;
                sqlComm.Connection = sqlConn;
                sqlComm.CommandText = procName;
                sqlComm.CommandType = System.Data.CommandType.StoredProcedure;
            }
            catch (Exception ex)
            {
                throw ex;
            }
        }
        #endregion
    }
}

File: BroadcastForClub/PegionClockingSMS/DAL/DatabaseConnection.vb
Option Explicit On
Option Strict On
Imports System.IO
Imports System.Windows.Forms
Public Class DatabaseConnection
    Public sqlconn As New SqlClient.SqlConnection
    Public sqlcomm As New SqlClient.SqlCommand
    Private servername As String
    Private databasename As String
    Private username As String
    Private password As String

    Private Sub ReadConnecntionStringFile()
        Dim sysdir As String = ""
        sysdir = My.Application.Info.DirectoryPath

        Dim connectionstring As String = sysdir & "\ConnectionString.inf"

        servername = "198.38.94.72"
        username = "sa"
        password = "06242009"

        If File.Exists(connectionstring) Then
            Using tr As TextReader = New StreamReader(connectionstring)
                databasename = "pigeonclocking_" + tr.ReadLine
            End Using
        End If
    End Sub



    Public Sub DatabaseConnection(ByVal procname As String)
        Me.ReadConnecntionStringFile()
        sqlconn.ConnectionString = "Address=" + servername _
                                    + ";database=" + databasename _
                                    + ";user id=" + username _
                                    + ";pwd=" + password
        Try
            sqlcomm.Connection = sqlconn
            sqlcomm.CommandType = CommandType.StoredProcedure
            sqlcomm.CommandText = procname
        Catch ex As Exception
            MessageBox.Show("Connection to the Server Failed" & Chr(13) & "Please call your System Developer for assistant", _
                            "Connection Error", MessageBoxButtons.OK, MessageBoxIcon.Information)
        End Try

    End Sub
End Class
